tests/unit_tests/test_helm_chart.py

```diff
@@ -0,0 +1,307 @@
+import subprocess
+from collections.abc import Iterable, Mapping
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+from textwrap import dedent
+from typing import Any, Literal
+
+import pytest
+import yaml
+from pydantic import Secret, TypeAdapter
+
+from blueapi.config import (
+    ApplicationConfig,
+    LoggingConfig,
+    OIDCConfig,
+    ScratchConfig,
+    ScratchRepository,
+    StompConfig,
+)
+
+BLUEAPI_HELM_CHART = Path(__file__).parent.parent.parent / "helm" / "blueapi"
+Values = Mapping[str, Any]
+ManifestKind = Literal[
+    "ConfigMap",
+    "Ingress",
+    "Service",
+    "StatefulSet",
+    "Deployment",
```

Comment:
```suggestion
    "Pod",
```

We do not produce any Deployments, but can make Pods from the test-connection